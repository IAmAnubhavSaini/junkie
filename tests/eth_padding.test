#!/bin/sh
# Check that TCP payload is 0 in the presence of ethernet padding
NAME="Ethernet Padding"
PCAPS="pcap/misc/eth_padding.pcap"
filter() {
	grep '^TCP' | cut -d' ' -f 2-
}
OUTPUT="eth_padding.out"

. ./dump_test
