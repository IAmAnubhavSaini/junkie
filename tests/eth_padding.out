-- pcap/misc/eth_padding.pcap --
head_len=20, payload=0, ports=993->1979, flags=Ack, win=65535, ack=2440074829, seq=1633020171
